Makefile

```diff
@@ -9,7 +9,7 @@ run: templating
 	touch $@
 
 templating: .bindata
-	@cd atlas/ && go generate
+	@cd atlas/ && rm -f templates/template-bindata.go && go generate
```

Comment:
is this required to remove files before generating new?

Reply:
After executing the `atlas init-app` command, we get the template-bindata.go file. After adding a certain amount of helm charts, this file began to update incorrectly. Therefore, the execution of the `atlas init-app` command hung. If the file is deleted, then it is created correctly.
I don’t know why this is happening. I made a workaround for the successful execution of the command.